src/storage/heap_file.c

```diff
@@ -8781,26 +8774,25 @@ heap_get_mvcc_header (THREAD_ENTRY * thread_p, const OID * oid, const OID * forw
  * int ispeeking (in) : PEEK or COPY.
  */
 static SCAN_CODE
-heap_get_record_data_when_all_ready (THREAD_ENTRY * thread_p, const OID * oid, OID * forward_oid, PAGE_PTR home_page,
-				     PAGE_PTR forward_page, INT16 record_type, RECDES * recdes,
+heap_get_record_data_when_all_ready (THREAD_ENTRY * thread_p, HEAP_GET_CONTEXT * context,
```

Comment:
@andrei14vl 

Would you also update its function comment with its new arguments?